setup.cfg

```diff
@@ -57,13 +57,10 @@ install_script = scripts/rpm-install.sh
 
 [flake8]
 exclude = build,.git,.tox,./tests/.env
-ignore = W504,W601
-max-line-length = 119
+extend-ignore = E203
```

Comment:
We need to ignore `W601` because cache backends have `has_key()` methods.

Reply:
Any chance we can ignore it in a single file?

Reply:
Yes we can :) I switched to `per-file-ignores` and bumped `flake8` to 3.7.0+.